### PR TITLE
Add block-building review to chaincode-labs/chaincode-podcast

### PR DESCRIPTION
**Pull request by markon1-a**
This PR adds [block-building](https://podcasters.spotify.com/pod/show/chaincode/episodes/Block-Building-with-Clara-and-Murch---Episode-18-e1dmitu) transcript review to the chaincode-labs/chaincode-podcast directory.

### CODE REVIEW

**Review by realdezzy — COMMENTED** [review]
Hey @markon1-a! Thank you for your diligent and consistent contributions to this great project.
I made  minor corrections in this commit [here](https://github.com/bitcointranscripts/bitcointranscripts/pull/483/commits/c8e947a3aba0d4213216e2db518d90dec4251450)

Review the commit and comments to understand the required changes.

**Comment by markon1-a** [comment]
Seen the changes made. Thanks.

File: chaincode-labs/chaincode-podcast/block-building.md [hunk 1]
```diff
@@ -1,47 +1,54 @@
...
-Speaker 0: 00:00:00
+Patrick Murck: 00:00:00
 
 The idea is we want the set of miners to be an open set where anybody can enter and exit as they wish.
-And if we now had obvious optimizations that people had to implement to be as competitive as possible, That would make it harder for new miners to enter this place.
-Hey, Murch.
+And if we now had obvious optimizations that people had to implement to be as competitive as possible, that would make it harder for new miners to enter this place.
+
 
 Speaker 1: 00:00:32
```
Comment (realdezzy) [inline chaincode-labs/chaincode-podcast/block-building.md:24]: I observed that the host's name has not been attributed. Could you please address this omission?
Please take time to familiarize yourself with hosts to help you in future reviews.
The name of the host for this current podcast is "Adam Jonas"

File: chaincode-labs/chaincode-podcast/block-building.md [hunk 2]
```diff
@@ -207,94 +213,96 @@ You could have an invalid script or invalid signature, or you could have a trans
...
 
-Speaker 2: 00:09:46
+Clara Shikhelman: 00:09:46
 
 So how would they do that?
 
-Speaker 0: 00:09:48
+Patrick Murck: 00:09:48
 
 Well, currently we assume that most miners are using Bitcoin Core and Bitcoin Core has a function called get block template.
```
Comment (realdezzy) [inline chaincode-labs/chaincode-podcast/block-building.md:235]: You missed this "get block template".
`getblocktimestamp` is a technical term and should be enclosed in backticks to align with our transcription style.
Please review the transcript for these AI errors and correct them

**Review by markon1-a — COMMENTED** [review]
Seen the crucial changes made.

**Comment by markon1-a** [comment]
Hello is there something have not changed?

**Comment by realdezzy** [comment]
> Hello is there something have not changed?

Sorry for the delayed response, we will take a look and give a response shortly

**Pull request merged**